### PR TITLE
Fixed yarn, babel and other problem which occured after dependencies version bump

### PR DESCRIPTION
**Pull request by BaderBC**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -118,3 +118,6 @@ keystore.json
 
 # Other package managers
 pnpm-lock.yaml
+
+# IDE's
+.idea/
```
Comment (maxidragon) [inline .gitignore:123]:
```suggestion
.idea/
.vscode/
```

**Comment by BaderBC** [comment]
Before merge I want @maxidragon to check if library fully works on his `HackHeroes2023` project

**Comment by maxidragon** [comment]
I'll check it and let you know

**Review by maxidragon — APPROVED** [review]
It works ;)

**Pull request merged**